components/grid/filter/overview.md

```diff
@@ -9,43 +9,43 @@ previous_url: /components/grid/filtering
+The FilterMenu filter mode renders a button in the column header. Clicking the button opens a popup with filtering options, allowing you to apply two filter criteria, choose a filter operator, and use buttons to apply or clear the filter. Read more about enabling and fine-tuning the filtering menu in the [FilterMenu documentation article...]({%slug grid-filter-menu%})
 
-The behavior of the filter input and the available filter operators will depend on the column data type. For example, a boolean field will only have the options "is true" and "is false" and will not have operators like "contains" or "greater than".
+> You can prevent the user from filtering a certain field by setting `Filterable="false"` on its column.
 
-You can filter more than one column at a time, and all filter rules will be applied together with an `AND` logic.
+## More Filtering Options
 
-You can prevent the user from filtering a certain field by setting `Filterable="false"` on its column.
+In addition to the two main filtering modes, the grid offers two more features that can enhance the user experience when looking for data.
 
-## More Filtering Options
+### ToolBar SearchBox
 
-In addition to the two main filtering modes, the grid offers two more features that can enhance the user experience when looking for data:
+The ToolBar of the Telerik Grid for Blazor includes a [SearchBox]({%slug grid-searchbox%}), that lets users amend filters and search across multiple fields simultaneously.
 
-* A [searchbox in the toolbar]({%slug grid-searchbox%}) can amend the filters and let the user look up many fields at once
+### CheckBoxList
 
-* The filter menu can show a [list of checkboxes]({%slug grid-checklist-filter%}) with the distinct values from the data to make filtering resemble Excel.
+The filter menu can display a [list of checkboxes]({%slug grid-checklist-filter%}) with distinct values from the data, making filtering similar to Excel.
 
 ## Filter Descriptors
 
-The Grid filter state is stored in [CompositeFilterDescriptors](/blazor-ui/api/Telerik.DataSource.CompositeFilterDescriptor). The below information is important if you want to [get or change the Grid filters programmatically]({%slug grid-state%}).
+The Grid filter state is stored in [CompositeFilterDescriptors](/blazor-ui/api/Telerik.DataSource.CompositeFilterDescriptor). Use the following information if you want to [get or change the Grid filters programmatically]({%slug grid-state%}).
 
-Each `CompositeFilterDescriptor` contains a [**collection** of `FilterDescriptor`s](/blazor-ui/api/Telerik.DataSource.FilterDescriptorCollection). All descriptors in the collection are applied with an *AND* or an *OR* `LogicalOperator`.
+Each CompositeFilterDescriptor includes a collection of FilterDescriptors, where all descriptors in the collection are applied with an AND or OR LogicalOperator.
```

Comment:
Don't invent new words when referring to well-known concepts&mdash;follow the common writing standards.
```suggestion
Each `CompositeFilterDescriptor` includes a collection of filter descriptors, where all descriptors in the collection are applied with an AND or OR logical operator.
```